ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -412,12 +659,31 @@ class FamilyReads extends React.PureComponent {
             />
           }
           { rnaTrackOptions &&
-            <CheckboxGroup
-              groupLabel="RNA Tracks"
-              value={this.state.sampleTypes}
-              options={rnaTrackOptions}
-              onChange={this.updateSampleTypes}
-            />
+            <div>
+              <CheckboxGroup
+                groupLabel="RNA Tracks"
+                value={this.state.sampleTypes}
+                options={rnaTrackOptions}
+                onChange={this.updateSampleTypes}
+              />
+              { this.state.sampleTypes.find(sampleType => rnaTrackOptions.find(opt => opt.value === sampleType)) &&
```

Comment:
I think it might be faster overall if you define a `RNA_TRACK_TYPE_LOOKUP` Set or dict so you can performantly check if an RNA sample type is selected instead of needing to always do a nested list iteration. You could also then update the check in `updateSampleTypes` as well. So something like 
```
const RNA_TRACK_TYPE_LOOKUP = new Set(... RNA_TRACK_TYPE_OPTIONS.map(({ value }) => value)
...
{this.state.sampleTypes.some(sampleType => RNA_TRACK_TYPE_LOOKUP.has(sampleType)) &&
```